Makefile

```diff
@@ -70,3 +70,33 @@ build-dep-fedora:
 	sudo dnf -y install --best --allowerasing --setopt=install_weak_deps=False \
 		--exclude=mariadb-server "$(MYSQL_SERVER_PACKAGE)"
 
+
+.PHONY: install-ubuntu
+install-ubuntu:
+	scripts/install-python-version $(PYTHON_VERSION)
+	sudo scripts/remove-default-mysql
+	sudo scripts/install-mysql-packages $(MYSQL_VERSION)
+	sudo scripts/setup-percona-repo
+	sudo scripts/install-percona-package $(PERCONA_VERSION)
+	sudo scripts/install-python-deps $(PYTHON_VERSION)
+	sudo scripts/create-user
+
+# local development, don't use in CI
+# prerequisite
+.PHONY: build-setup-specific-image
+build-setup-specific-image:
+	PYTHON_VERSION=$(PYTHON_VERSION) MYSQL_VERSION=$(MYSQL_VERSION) PERCONA_VERSION=$(PERCONA_VERSION) scripts/build-setup-specific-test-image
+
+.PHONY: dockertest
+dockertest:
+	docker run -it --rm myhoard-test-temp /src/scripts/test-inside
+
+# when the image didn't change this can be used. local dev only, don't use in CI
+# in this target we override the /src that gets used to rsync source inside the container
+.PHONY: dockertest-resync
+dockertest-resync:
+	docker run -it --rm -v "$(shell pwd):/src:ro" myhoard-test-temp /src/scripts/test-inside
+
+.PHONY: dockertest-pytest
+dockertest-pytest:
+	docker run -it --rm -v "$(shell pwd):/src:ro" myhoard-test-temp /src/scripts/pytest-inside $(PYTEST_ARGS)
```

Comment:
Missing newline at end of file.

Reply:
Do we have any standard about this? I'd feel silly to check and fix this manually. ```make lint``` doesn't catch this either.

Reply:
pylint does it for python files (C0304), but we don't have a linter for non-python.
the standard is... [posix](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) :D (and GitHub showing a red sign on the line)

Most text editors and IDEs have an option to fix that automatically instead of waiting for a linter:
 - Pycharm: Settings→Editor→General→Ensure every saved files ends with a line break (at the bottom)
 - VsCode: File→Preferences→Settings→Search for "insert final newline"
 - VIM: `:set fixendofline`

Reply:
Yeah, I know that it's posix, but I'm not sure we should focus on this kind of details in a review. This should be catched (and fixed!) by some automatic tool. I'll change those right now, btw.